### ISSUE TITLE
Spezielle Rechte für Kunden (KEINE ADMINS!)

### ISSUE THREAD

**Issue by vogelor** [mittwald/feature-requests#57]
**Welches Problem möchtest du lösen? Wann tritt es auf?**
Weniger Rechte für Kunden.
Wenn z.B. ein Kunde ein E-Mail Postfach anlegt, kann er es auf "unbegrenzt" setzen.
Das will ich aber nicht, ich will z.B. 10 Kunden mit je 10GB auf eine 50GB Platte "legen".

**Welche Lösungsideen hast du?**
Es sollte eine Rechtegruppe "Kunden" geben, die halt weniger kann.
Keine Postfachgröße usw.

**Hast du zusätzliche Informationen (wie z.B. Screenshots)?**

**Comment by paul-hph** [comment]
https://github.com/mittwald/feature-requests/issues/40

**Comment by vogelor** [comment]
Ja, aber nicht nur Limits, sondern evtl. gar nicht erst die Möglichkeit, etwas zu setzen.
D.h.  nicht nur, 2GB Pro Postfach, sondern gar nicht die Größe ändern können.
Ode bei anderen Dingen, die eher "Administrativ" sind gar nicht die Möglichkeit geben, es zu tun...

**Comment by paul-hph** [comment]
Im "alten" Sytem gibt es Berechtigungsübersicht wo man zumindest die Option "Bearbeiten bearbeiten" entfernen konnte. 

![image](https://github.com/mittwald/feature-requests/assets/3603487/1891896b-0786-4507-9813-413815e65685)

**Comment by patrickhilker** [comment]
Möglicherweise Überschneidungen mit https://github.com/mittwald/feature-requests/issues/82
